test/test_all_basic.rb

```diff
@@ -7,12 +7,12 @@
 FLOWER_HAT = IMAGES_DIR + '/Flower_Hat.jpg'
 IMAGE_WITH_PROFILE = IMAGES_DIR + '/image_with_profile.jpg'
 
+require 'simplecov'
 require 'test/unit'
 if RUBY_VERSION < '1.9'
```

Comment:
Can we also remove this `if` check? The `require` was inside that block and 1.9 is no longer supported.

Reply:
it is other thing. I dislike that one PR contains the multiple purpose.

Reply:
I also don't like combing to many things in one PR but I think this one is in a gray area so I would include that myself but I would also understand it if you would move this to a separate PR.